TFNetworkLayer.py

```diff
@@ -4113,6 +4113,7 @@ def transform_config_dict(cls, d, network, get_layer):
   def get_out_data_from_opts(cls, true_from, **kwargs):
       return get_concat_sources_data_template([true_from], name="%s_output" % kwargs["name"])
```

Comment:
Btw, this is overkill. You are not concatenating here. You could just do `true_from.output.copy(name="%s_output" % kwargs["name"])`.